conda/dev.yml

```diff
@@ -26,10 +26,9 @@ dependencies:
   # If versions are updated, also update in `.github/workflows/build_workflow.yml`
   # =================
   - sphinx=5.2.3
+  - sphinx-multiversion=0.2.4
```

Comment:
I don't think this matters but using single rather than double `=` here to match other dependencies.